howto/manage/web-dashboard.md

```diff
@@ -74,3 +74,37 @@ You can select any application you previously created and start a streaming sess
 ![Start a streaming session|690x352](https://assets.ubuntu.com/v1/6087fbd9-gs_dashboard_start_session.png)
 
 For more information about the Streaming Stack, visit [About application streaming](https://discourse.ubuntu.com/t/streaming-android-applications/17769).
+
+#### Streaming statistics
+
+View the streaming statistics for your running sessions by selecting the **Statistics** button on the session. The statistics display on the right pane and also have a download option to download the statistics in a `.csv` format for further analysis.
+
+The downloaded `.csv` file has the following statistics:
+
+| Statistic | Description |
+| --------- |------------ |
+| `date` | Date and time in ISO format |
+| `network-currentrtt` | Current round-trip time of the network in milliseconds |
+| `video-bandwidth` | The amount of video data that the session can handle per second |
+| `video-totalreceived` | Total video data received |
+| `video-fps` | Video frames per second |
+| `video-decodetime` | Time taken to extract video |
+| `video-jitter` | Loss of transmitted video data during streaming |
+| `video-averagejitterbufferdelay` | Average jitter buffer delay in video transmission  |
+| `video-packetsreceived` | Number of video packets received |
+| `video-packetslost` | Number of video packets lost |
+| `audio-bandwidth` | The amount of audio data that the session can handle per second |
+| `audio-totalreceived` | Total audio data received during streaming |
+| `audio-totalsamplesreceived` | Number of audio samples received |
+| `audio-jitter` | Loss of transmitted audio data during streaming |
+| `audio-averagejitterbufferdelay` | Average jitter buffer delay in audio transmission |
+| `audio-packetsreceived` | Number of audio packets received |
+| `audio-packetslost` | Number of audio packets lost |
```

Comment:
As a general comment: should we add the unit types for each statistic?
e.g.,
| `video-decodetime` | Time taken to extract video in milliseconds |
...
| `audio-totalreceived` | Total audio data received during streaming in bytes|

What do you think?

Reply:
Actually we use MB for "total received". I'm attaching a CSV with the new format, which has the unit in parentheses as Keirthana suggested.

[cishd4r4ffvkpp5g28h0_stats-history.csv](https://github.com/anbox-cloud/discourse-docs/files/12376989/cishd4r4ffvkpp5g28h0_stats-history.csv)

Reply:
Thanks @lorumic! Actually, since we have the unit in the CSV, we don't have to state it again in the documentation. The documentation is mainly for users to know how to deal with what is in the CSV file. I will remove the unit for the  `network-currentrtt` as well to be consistent.